scipy/spatial/ckdtree.pyx

```diff
@@ -876,7 +896,7 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
         --------
         >>> from scipy import spatial
         >>> x, y = np.mgrid[0:4, 0:4]
-        >>> points = zip(x.ravel(), y.ravel())
+        >>> points = list(zip(x.ravel(), y.ravel()))
```

Comment:
Is there a better number construct for doing this? I see you used numpy.c_ later.